xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/macros.vm

```diff
@@ -1767,6 +1773,9 @@ function (row, i, table) {
         if (fieldName === 'doc_name' && !row['doc_viewable']) {
           text += '*';
         }
+        if (fieldName === 'doc_title' && row['doc_title_computed']) {
+          text += '\u00b9';
```

Comment:
I think it would be nicer to add a CSS class on the ``container`` (table cell). This would allow us to apply whatever styling we want, including adding the superscript 1 using the ``:after`` selector. This is important especially since we could have titles that end with ``\u00b9``.

Reply:
If I add a CSS class to the container, then maybe I should try to add the same class to the explanatory message so they do not end up differerent symbols.
I will give it a try to see if that is possible without needing awkward code.